### PR TITLE
Update Preparing release in contributing docs

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
@tjanez Please review.

File: docs/contributing.rst [hunk 1]
```diff
@@ -60,6 +60,13 @@ Prepare Resolwe for development::
 
 .. _Resolwe's git repository: https://github.com/genialis/resolwe
 
+Signed commits are required in the Resolwe upstream repository.
```
Comment (tjanez) [inline docs/contributing.rst]: I would move this to a separate section named **Submitting changes to upstream** and put it just before the **Preparing release** section.

File: docs/contributing.rst [hunk 2]
```diff
@@ -60,6 +60,13 @@ Prepare Resolwe for development::
...
+Install your personal `GPG key`_ and
```
Comment (tjanez) [inline docs/contributing.rst]: Install -> Generate

File: docs/contributing.rst [hunk 3]
```diff
@@ -132,29 +133,36 @@ Commit changes to git::
...
+Wait for the tests to pass and the pull request to be approved. Merge
+the code to master::
 
-    twine upload dist/*
+    git checkout master
+    git merge --ff-only release-<new-version>
+    git push <resolwe-upstream-name> master <new-version>
 
-Tag the new version and sign it with your `GPG key`_::
+Tag the new stable version (we need to checkout the second to last
```
Comment (tjanez) [inline docs/contributing.rst]: I would replacing the following 4 lines with:

```
Tag the new release from the penultimate commit::

    git checkout master~1
    git tag -m "Version <new-version>" <new-version>
```

File: docs/contributing.rst [hunk 4]
```diff
@@ -132,29 +133,36 @@ Commit changes to git::
 
     git commit -a -m "Prepare release <new-version>"
 
-Test the new version with Tox_::
-
-    tox -r
+Bump project's version in ``resolwe/__about__.py`` file to the next
```
Comment (tjanez) [inline docs/contributing.rst]: Please wrap the lines at 80 (79) characters, not before.

Applies for all changes.

**Comment by mstajdohar** [comment]
All fixed. Please review again.

**Pull request merged**